include/z64save.h

```diff
@@ -354,13 +354,13 @@ typedef enum {
 #define EVENTCHKINF_67 0x67
 #define EVENTCHKINF_68 0x68
 #define EVENTCHKINF_69 0x69
-#define EVENTCHKINF_6A 0x6A
+#define EVENTCHKINF_TALON_WOKEN_IN_KAKARIKO 0x6A
 
 // 0x6B
-#define EVENTCHKINF_6B_INDEX 6
-#define EVENTCHKINF_6B_SHIFT 11
-#define EVENTCHKINF_6B_MASK (1 << EVENTCHKINF_6B_SHIFT)
-#define EVENTCHKINF_6B ((EVENTCHKINF_6B_INDEX << 4) | EVENTCHKINF_6B_SHIFT)
+#define EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO_INDEX 6
+#define EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO_SHIFT 11
+#define EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO_MASK (1 << EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO_SHIFT)
+#define EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO 0x6B
```

Comment:
```suggestion
#define EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO ((EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO_INDEX << 4) | EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO_SHIFT)
```
the idea with this kind of thing is that for moving the flag you would then only have to touch the index, shift, or both, instead of possibly this define and index/shift going out of sync, which would most likely cause bugs

but I'm biased in defending this approach because I came up with this so if someone else thinks we shouldn't bother, feel free to say

Reply:
i'm of the opinion that the index/shift macros are annoyingly tedious, and the effort would be better spent by making a custom getter and check macros for this special case. Either something more generic like

    #define GETVAL_EVENTCHKINF(flag) (gSaveContext.eventChkInf[(flag) >> 4])
    #define CHECKVAL_EVENTCHKINF(v, flag) ((v) & (1 << ((flag) & 0xF)))

Reply:
Well this is outside the scope of this PR, especially since they're a new contributor. Feel free to bring up this discussion on the discord

Reply:
I don't really have any opinion about this; maybe it would be clearer if z64save.h only had the flags as simple indices, because most of the flags follow that format already, and `En_Hy` handled the shifting and stuff with the macros suggested by @mzxrules. 
But yeah, that should probably be done later when someone documents `En_Hy` (afaik that's the only place where `_INDEX` and `_MASK` are used).

Should I restore `EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO` to just be `0x6B`?

Reply:
Well for now and for consistency this should stay _index and _shift since other flags do that

If this is changed down the line everything will be changed at once